src/realm/alloc_slab.cpp

```diff
@@ -806,6 +807,20 @@ ref_type SlabAlloc::attach_file(const std::string& file_path, Config& cfg)
                 realm::util::encryption_read_barrier(map_header, 0, sizeof(Header));
             }
         }
+        if (top_ref) {
+            // Check if file size can be reduced
+            constexpr size_t file_size_ndx = 2;
```

Comment:
Isn't this constant already declared somewhere? Or is it not in scope here?

Reply:
Added a comment that it must match definition in Group